eng/common/tools.ps1

```diff
@@ -498,7 +498,7 @@ function InitializeXCopyMSBuild([string]$packageVersion, [bool]$install) {
 #       Two part minimal VS version, e.g. "15.9", "16.0", etc.
 #   "components": ["componentId1", "componentId2", ...]
 #       Array of ids of workload components that must be available in the VS instance.
-#       See e.g. https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2017
+#       See e.g. https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise
```

Comment:
```suggestion
#       See e.g. https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise
```

Reply:
I can change this, but I was told by dougbu to leave the links in this directory alone because they are changed automatically.

> Undo all changes under eng/common/; these files are overridden every time the Arcade SDK is updated.